rero_ils/modules/cli.py

```diff
@@ -47,11 +48,19 @@
 from .items.cli import create_items, reindex_items
 from .loans.cli import create_loans
 from .patrons.cli import import_users
-from .providers import append_pids_to_table
 
 _datastore = LocalProxy(lambda: current_app.extensions['security'].datastore)
 
 
+def append_fixtures_new_identifiers(table, pids):
```

Comment:
Do not understand why this function was moved from `provider` to `cli`

Reply:
moved back and test created.